_posts/2024-09-02-geoserver-2-26-M0-released-for-A-B-testing.md

```diff
@@ -0,0 +1,101 @@
+
+One of the frameworks that GeoServer is built upon is [Apache Wicket](https://wicket.apache.org/), and it needs to be upgraded to a later, supported version.  This is a substantial change that potentially affects every GUI (graphical user interface) page, so we are asking the **GeoServer community** to review each GUI page to check if there are any failures that need to be handled manually.  As a user, you are the very best people to do this testing, as you are most familiar with the GUI, and it's a great way for you to relieve some of the pressure on the over-worked developers.  **Here's how you can help**:
+
+## Every GUI page!?!
+
+You do not need to review every single page, if the task is broken down into bite-sized chunks and split across multiple testers!  That's what [this spreadsheet](https://docs.google.com/spreadsheets/d/1pQmncG4zxpgJnHxeI4myFfOBD17U2CIMcy59II4XAfo/edit) intends to coordinate.  Please pick pages that are familiar to you and that have not already been checked by another helpful volunteer.  Update the spreadsheet to indicate that you have tested a page and document any strange results for a developer to investigate further.
+
+## How do I spot the differences?
+
+We would like you to compare the new milestone 2.26-M0 version against the latest stable version (2.25.3), by opening 2 browser tabs and performing the same action in each tab, to see if the GUI is identical.  It should look the same; if not, please indicate on the testing spreadsheet for someone else to double check.
+
+## How do I run the new milestone 2.26-M0 version?
+
+### Docker
+
+If you (or your DevOps team) are already using a Docker image with your own data, that's wonderful, you're half way there.  (`docker pull docker.osgeo.org/geoserver:2.26-M0`)
+
+If not, we have packaged 2.26-M0 as a Docker image as the most portable way to distribute the task to you.  You need Docker installed ([https://docs.docker.com/get-started/get-docker/](https://docs.docker.com/get-started/get-docker/)) and then you need to configure it to run with the extensions and community modules that you're familiar with.  Even more preferable would be if you can also run it using your own data.  You should refer to the configuration instructions in the [README.md](https://github.com/geoserver/docker)
+
+_BTW, this is also a great opportunity, if you have been thinking about it, for your company to make the switch to a containerised version - we're here to assist you with getting started, if you assist us with the testing!  Also, once you have set up Docker and your own data, you will be in a fine position to respond to our pleas to test release candidates (both major and minor) with your own data, prior to the release being made - so that you know it will work for you!  Win-win!_
+
+Here's a simple example to work from:
+
+```
+  docker run -p8081:8080 --rm --name geoserver-2-26-M0 \
+	--mount src="my-local-drive/my-GEOSERVER_DATA_DIR",target=/opt/geoserver_data/,type=bind \
```

Comment:
this  (``) emoticon is not rendering properly, did you mean to use `$GEOSERVER_DATA_DIR`?

Reply:
I did not mean to use `$`, but I certainly did not mean to include whatever that character is.  Thanks for spotting it!